tests/acceptance/features/bootstrap/TUSContext.php

```diff
@@ -92,42 +107,57 @@ public function createNewTUSResourceWithHeaders(string $user, TableNode $headers
 	public function createNewTUSResource(string $user, TableNode $headers): void {
 		$rows = $headers->getRows();
 		$rows[] = ['Tus-Resumable', '1.0.0'];
-		$this->createNewTUSResourceWithHeaders($user, new TableNode($rows));
-		$this->featureContext->theHTTPStatusCodeShouldBe(201);
+		$response = $this->createNewTUSResourceWithHeaders($user, new TableNode($rows));
+		$this->featureContext->theHTTPStatusCodeShouldBe(201, "", $response);
 	}
 
 	/**
-	 * @When /^user "([^"]*)" sends a chunk to the last created TUS Location with offset "([^"]*)" and data "([^"]*)" using the WebDAV API$/
-	 *
 	 * @param string $user
 	 * @param string $offset
 	 * @param string $data
 	 * @param string $checksum
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 *
 	 * @throws GuzzleException
 	 * @throws JsonException
 	 */
-	public function sendsAChunkToTUSLocationWithOffsetAndData(string $user, string $offset, string $data, string $checksum = ''): void {
+	public function sendsAChunkToTUSLocationWithOffsetAndData(string $user, string $offset, string $data, string $checksum = ''): ResponseInterface {
 		$user = $this->featureContext->getActualUsername($user);
 		$password = $this->featureContext->getUserPassword($user);
-		$this->featureContext->setResponse(
-			HttpRequestHelper::sendRequest(
-				$this->resourceLocation,
-				$this->featureContext->getStepLineRef(),
-				'PATCH',
-				$user,
-				$password,
-				[
-					'Content-Type' => 'application/offset+octet-stream',
-					'Tus-Resumable' => '1.0.0',
-					'Upload-Checksum' => $checksum,
-					'Upload-Offset' => $offset
-				],
-				$data
-			)
+		$response = HttpRequestHelper::sendRequest(
+			$this->resourceLocation,
+			$this->featureContext->getStepLineRef(),
+			'PATCH',
+			$user,
+			$password,
+			[
+				'Content-Type' => 'application/offset+octet-stream',
+				'Tus-Resumable' => '1.0.0',
+				'Upload-Checksum' => $checksum,
+				'Upload-Offset' => $offset
+			],
+			$data
 		);
+		return $response;
+	}
+
+	/**
+	 * @When /^user "([^"]*)" sends a chunk to the last created TUS Location with offset "([^"]*)" and data "([^"]*)" using the WebDAV API$/
```

Comment:
if you want to make it easier to read
```suggestion
	 * @When user :user sends a chunk to the last created TUS Location with offset :offset and data :data using the WebDAV API$/
```